viahtml/wsgi.py

```diff
@@ -3,8 +3,7 @@
 import os
 import signal
 
-# Our job here is to leave this `application` attribute laying around as
-# it's what uWSGI expects to find.
```

Comment:
This comment was causing a fight between black and isort. When you run `make format` black inserts a blank line before the comment then isort removes the blank line, then you run `make checkformatting` or `make sure` black complains that the blank line is missing.